### PR TITLE
added a bunch of cards

### PR DESCRIPTION
**Pull request by aoaaceai**

### CODE REVIEW

File: src/main/java/eod/card/concrete/summon/GuerrillaShooterSummon.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    @Override
+    public String getName() {
+        return "召喚 游擊隊槍手";
+    }
+
+    @Override
+    public Party getParty() {
+        return null;
```
Comment (secminhr) [inline src/main/java/eod/card/concrete/summon/GuerrillaShooterSummon.java]: It's RED.

File: src/main/java/eod/event/ObjectMovingEvent.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package eod.event;
+
+import eod.warObject.WarObject;
+
+import java.awt.*;
+
+public class ObjectMovingEvent implements Event {
+    private WarObject object;
+    Point origPos, newPos;
```
Comment (secminhr) [inline src/main/java/eod/event/ObjectMovingEvent.java]: Please add an access modifier.

File: src/main/java/eod/specifier/condition/InAttackRangeCondition.java [hunk 3]
```diff
@@ -4,12 +4,20 @@
 import eod.warObject.CanAttack;
 import eod.warObject.WarObject;
 
+import java.awt.*;
+import java.util.ArrayList;
 import java.util.Arrays;
 
 public class InAttackRangeCondition implements Condition{
     WarObject center;
+    ArrayList<Point> attackRange;
```
Comment (secminhr) [inline src/main/java/eod/specifier/condition/InAttackRangeCondition.java]: Access modifier

File: src/main/java/eod/warObject/character/concrete/red/GuerrillaShooter.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import eod.warObject.Damageable;
+import eod.warObject.character.abstraction.assaulter.Shooter;
+
+import java.awt.*;
+import java.util.ArrayList;
+
+import static eod.effect.EffectFunctions.RequestRegionalAttack;
+
+public class GuerrillaShooter extends Shooter {
+    int specialEffectTimes = 0;
```
Comment (secminhr) [inline src/main/java/eod/warObject/character/concrete/red/GuerrillaShooter.java]: access modifier

**Review by secminhr — COMMENTED** [review]
Comments

**Comment by aoaaceai** [comment]
fixed

**Review by secminhr — APPROVED** [review]
good work1!!!!!!!!!

**Pull request merged**